recipes/netcdf/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 2.8.0 FATAL_ERROR)
```

```suggestion
cmake_minimum_required(VERSION 3.1)
```

Targets requires CMake 3+